bigframes/session/loader.py

```diff
@@ -727,6 +729,7 @@ def _query_to_destination(
         api_name: str,
         configuration: dict = {"query": {"useQueryCache": True}},
         do_clustering=True,
+        max_results: Optional[int] = None,
```

Comment:
Ended up using a different workaround (renamed _TABLE_SUFFIX to _BF_TABLE_SUFFIX), so we can revert this change.